inc/saiswitch.h

```diff
@@ -1213,10 +1311,8 @@ typedef sai_status_t (*sai_get_switch_attribute_fn)(
  */
 typedef struct _sai_switch_api_t
```

Comment:
Is it worth to update applications that uses SAI (i.e., PTF) in this PR? After this PR is merged, will PTF compile?